tsdb/index/tsi1/partition.go

```diff
@@ -897,6 +900,46 @@ func (p *Partition) compactionsEnabled() bool {
 	return p.compactionsDisabled == 0
 }
 
+func (p *Partition) runPeriodicCompaction() {
+	// kick off an initial compaction at startup without the optimization check
+	p.Compact()
+
+	// check for compactions once an hour (usually not necessary but a nice safety check)
```

Comment:
Doesn't this default to 4 hours, not one hour?

Reply:
Fixed to be hard-coded as hourly.